content-repo/gendocs.py

```diff
@@ -127,6 +127,13 @@ def get_deprecated_data(yml_data: dict, desc: str, readme_file: str):
     return ""
 
 
+def get_fromversion_data(yml_data: dict):
+    from_version = yml_data.get('fromversion', '')
+    if not from_version.startswith(('4', '5.0')):
```

Comment:
Also check that from_version is not empty

Reply:
done